capitulo 1/lang_en.json

```diff
@@ -2963,16 +2963,16 @@
   "obj_npc_room_slash_Other_10_gml_619_0": "* (Você está gostando do espetáculo?)/%",
   "obj_npc_room_slash_Other_10_gml_620_0": "* (Eu sou o Goulden Sam.)/%",
   "obj_npc_room_slash_Other_10_gml_621_0": "* (É difícil ser a quarta cela.)/%",
-  "obj_npc_room_slash_Other_10_gml_638_0": "* Cara^1! Ah^1, cara^1!&* Nós éramos os Antigos Reis^1!&* Nós te imploramos^1, nos liberte!/",
-  "obj_npc_room_slash_Other_10_gml_639_0": "* Uma vez^1, nós quatro reinávamos jutos^1, mas quando o Cavalheiro chegou, tudo mudou.../",
+  "obj_npc_room_slash_Other_10_gml_638_0": "* Angústia^1! Ah^1, Angústia^1!&* Nós éramos os Antigos Reis^1!&* Imploramos a você^1, nos liberte!/",
+  "obj_npc_room_slash_Other_10_gml_639_0": "* Uma vez^1, nós quatro reinávamos juntos^1, mas quando o Cavalheiro chegou, tudo mudou.../",
   "obj_npc_room_slash_Other_10_gml_640_0": "* Ah^1, meus diamantes^1! Minhas joias^1! Você tem que me dizer se elas estão bem^1! Ahh!!/%",
-  "obj_npc_room_slash_Other_10_gml_644_0": "* O que^1, você disse? Seu mundo está correndo perigo?/",
+  "obj_npc_room_slash_Other_10_gml_644_0": "* O que^1, disses? Seu mundo está correndo perigo?/",
   "obj_npc_room_slash_Other_10_gml_645_0": "* Bem^1, é uma pena^1, não é./%",
-  "obj_npc_room_slash_Other_10_gml_665_0": "* Eu fui aprisionado por fazer um enigma sem permissão.../",
+  "obj_npc_room_slash_Other_10_gml_665_0": "* Eu fui aprisionado por fazer um enigma sem licença.../",
   "obj_npc_room_slash_Other_10_gml_666_0": "* Eu..^1. Eu nunca mais vou fazer outro caça-palavras^1! Prometo!/%",
   "obj_npc_room_slash_Other_10_gml_653_0": "* Só porque eu não queria lutar com vocês caras^1, eu fui preso.../",
   "obj_npc_room_slash_Other_10_gml_654_0": "* Como pode!^1? Vocês parecem ser BEM melhores do que o Rei...!/%",
-  "obj_npc_room_slash_Other_10_gml_659_0": "* Haha^1, eu prefiro ficar nas prisão do que enfrentar vocês^1! Vocês são terríveis!/%",
+  "obj_npc_room_slash_Other_10_gml_659_0": "* Haha^1, eu prefiro ficar na prisão do que enfrentar vocês^1! Vocês são terríveis!/%",
```

Comment:
Como disse, eu tava com sono naquele momento e os resquícios da linguagem coloquial estavam mais fortes do que o normal. By the way, aprovado.

Reply:
Sim, é no sentido de serem bastante assustadores pra ele, só que acho que colocar _assustadores_ seria muito leve e amenizaria o medo dele. Já _medonho_, ninguém fala isso numa situação dessas kkk além de que medonho se referiria mais a aparência física deles quando não é esse o ponto que deixa o carinha ali com medo.

Reply:
Terrível confunde com falta de habilidade. Acho estranho. Os comuns são medonhos/macabros para dar mais ênfase, ou os normais como assustadores, horripilantes, aterrorizantes.